data/src/test/java/com/onboarding/data/CharacterMapperTest.kt

```diff
@@ -0,0 +1,83 @@
+package com.onboarding.data
+
+import com.onboarding.data.mapper.transform
+import com.onboarding.data.service.response.CharacterBaseResponse
+import com.onboarding.data.service.response.CharactersResultResponse
+import com.onboarding.data.service.response.InfoResponse
+import com.onboarding.data.service.response.LocationLinkResponse
+import com.onboarding.data.util.ConstantUtil.INT_VALUE
```

Comment:
The constants used in the tests should be declared in the test.
Do not use the ConstantUtil class, define the constants in the companion pls

Reply:
done